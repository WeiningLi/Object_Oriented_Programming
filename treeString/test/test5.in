ra "a"
rb "b"
+aab
cca
ib0c
+ccb
cdc
ic0'
+ddc
cad
pa
da
^a1 "?????"
pa
q
