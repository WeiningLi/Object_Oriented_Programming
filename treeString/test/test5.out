abc'bc
"c" size: 3
 "b" size: 2
  "a" size: 1
 "b" size: 2
  "'" size: 1
  "c" size: 1
a?????bc'bc
